graphql_api/types/repository/repository.py

```diff
@@ -283,6 +283,15 @@ def resolve_flags_measurements_backfilled(repository: Repository, info) -> bool:
     return dataset.is_backfilled()
 
 
+@repository_bindable.field("isATSConfigured")
+def resolve_is_ats_configured(repository: Repository, info) -> bool:
+    if not repository.yaml or "flag_management" not in repository.yaml:
+        return False
+
+    individual_flags = repository.yaml["flag_management"].get("individual_flags", {})
+    return individual_flags.get("carryforward_mode") == "labels"
```

Comment:
Id add a default value for `carryforward_mode` here, so `individual_flags.get("carryforward_mode", '')`

Reply:
`.get()` returns `None` by default so the code still executed as expected in this case.

Reply:
True that, I mostly try to give an explicit default to be explicit 😂 but this is okay